doc/userguide/rules/dns-keywords.rst

```diff
@@ -32,9 +50,10 @@ Match on DNS requests where the **opcode** is NOT 0::
 dns.query
 ---------
 
-With **dns.query** the DNS request queries are inspected. The dns.query
-keyword works a bit different from the normal content modifiers. When
-used in a rule all contents following it are affected by it.  Example:
+With **dns.query** the DNS request query names are inspected. The
+dns.query keyword works a bit different from the normal content
```

Comment:
this works a bit different bit could be removed. It's just about sticky buffer vs content modifier

Reply:
Given we're at 8 maybe we should clean this up. Just refer to it as a sticky buffer and not much more?